config/default/kustomization.yaml

```
@@ -12,31 +9,31 @@ namespace: capz-system
 namePrefix: capz-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+commonLabels:
+  cluster.x-k8s.io/provider: "infrastructure-azure"
 
-resources:
-- ../crd
-- ../rbac
-- ../manager
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
-#- ../webhook
-# [CERTMANAGER] To enable cert-manager, uncomment next line. 'WEBHOOK' components are required.
-#- ../certmanager
+bases:
+  - ../crd
+  - ../rbac
+  - ../manager
+  # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
+  #- ../webhook
+  # [CERTMANAGER] To enable cert-manager, uncomment next line. 'WEBHOOK' components are required.
+  - ../certmanager
```

```suggestion
  #- ../certmanager
```

Based on this comment, if we're enabling cert-manager, we need to uncomment the webhook sections here and in `config/crd/kustomization.yaml`. I would say let's defer that to another PR, unless there's a good reason to enable it here.

---

ok, will remove that and we can add it again in a different PR with the webhook